responsibleai_text/responsibleai_text/managers/explainer_manager.py

```diff
@@ -29,6 +29,23 @@
                                                  Tokens)
 from responsibleai_text.utils.question_answering import QAPredictor
 
+try:
+    from interpret_text.generative.lime_tools.explainers import \
+        LocalExplanationSentenceEmbedder
+except ImportError as e:
+    print("Could not import LocalExplanationSentenceEmbedder: ", e)
+
+try:
+    from interpret_text.generative.model_lib.openai_tooling import ChatOpenAI
+except ImportError as e:
+    print("Could not import ChatOpenAI: ", e)
```

Comment:
maybe we can instead set chat_openai_installed = False and then throw before it is used in line 175, and similar for other imports?
Like the import here:
https://github.com/microsoft/ml-wrappers/blob/47babf7185664cb25b47409f912aa7e601213444/python/ml_wrappers/model/openai_wrapper.py#L12
And exception here:
https://github.com/microsoft/ml-wrappers/blob/47babf7185664cb25b47409f912aa7e601213444/python/ml_wrappers/model/openai_wrapper.py#L178